Datasets/LabeledExample.swift

```diff
@@ -34,3 +34,17 @@ public struct LabeledExample: TensorGroup, KeyPathIterable, Collatable {
         data = Tensor<Float>(handle: TensorHandle<Float>(handle: _handles[dataIndex]))
     }
 }
+
+// A simple tuple of Tensor, useful for many deep learning tasks (one tensor of inputs,
+// one tensor of labels). Note that if your task is more elaborate, you should write your
+// own struct of Tensors. Automatic conformance to Collatable can be derived the same way as
+// here.
```

Comment:
It is very important to use backticks around type names in documentation comments, so that the type names will be formatted correctly in [generated API documentation](https://www.tensorflow.org/swift/api_docs).

Please use `/// - Note:` for "note" documentation blurbs. This helps clearly separate main documentation (what is `TensorPair`?) from auxiliary documentation. They also show up nicely in generated API documentation.

```suggestion
/// A generic tuple of two tensors.
///
/// - Note: `TensorPair` has a generic name and provides little semantic information. Rather than using this type, consider creating a custom type with a more descriptive name.
```

Reply:
Will do, not sure there is a documentation for swift-models yet?

Reply:
Good point, `swift-models` doesn't have hosted API docs yet.